PHP/ApiClients/CampaignApi.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+include_once 'ClientBase.php';
+include_once 'Http/CurlWrapper.php';
+include_once 'Model/RegisterPageRequest.php';
+include_once 'Model/StoryUpdateRequest.php';
+
+class CampaignApi extends ClientBase
+{		
+	public $Parent;
+	public $curlWrapper;
+	
+	public function __construct($justGivingApi)
+	{
+		$this->Parent		=	$justGivingApi;
+		$this->curlWrapper	= new CurlWrapper();
+	}
+
+	public function Retrieve($charityName, $campaignName)
+	{
+		$locationFormat = $this->Parent->RootDomain . "{apiKey}/v{apiVersion}/campaigns/". $charityName ."/". $campaignName;
+		$url = $this->BuildUrl($locationFormat);
+		$json = $this->curlWrapper->Get($url, $this->BuildAuthenticationValue());
+		return json_decode($json);
+	}
+
+	public function Create($campaignCreationRequest)
+	{
+		$locationFormat = $this->Parent->RootDomain . "{apiKey}/v{apiVersion}/campaigns";
+		$url = $this->BuildUrl($locationFormat);
+		$payload = json_encode($campaignCreationRequest);
+		$json = $this->curlWrapper->Put($url, $this->BuildAuthenticationValue(), $payload);
```

Comment:
Same issue as it was with Retrieve, it should be [PutV2](https://github.com/JustGiving/JustGiving.Api.Sdk/blob/master/PHP/ApiClients/AccountApi.php#L35)